packages/meteor-otel/tests/tracing_tests.js

```
@@ -0,0 +1,576 @@
+});
+
+Tinytest.add('meteor-otel - tracing - injectTraceContext adds traceparent when in span', (test) => {
+  let headers;
+
+  withSpanSync('test-tracer', 'test-span', () => {
+    headers = injectTraceContext();
+  });
+
+  test.isNotNull(headers);
+  // traceparent should be present when inside a span
+  if (headers.traceparent) {
```

ESLint

[error] Insert `··`

(prettier/prettier)